examples/statistics/confidence_ellipse.py

```diff
@@ -213,3 +214,21 @@ def get_correlated_dataset(n, dependency, mu, scale):
 
 fig.subplots_adjust(hspace=0.25)
 plt.show()
+
+#############################################################################
+#
+# No 68-95-99.7 Rule
+# """"""""""""""""""
+#
+# .. no_rule:
+# The well-known 68-95-99.7 rule states that in a normal distribution 68%
+# of the observations lie within +/- 1 standard deviation from the mean, 95%
+# within 2 standard deviations and 99.7% within 3 standard deviations.
+#
+# However this rule only holds for one-dimensional data.
+# To adjust this rule to two-dimensional data,
+# it suffices to square the probabilities
+# of 0.68, 0.95 and 99.7 to 0.462, 0.903 and 0.994, respectively.
+# If you wish, you can think of it as a new "46-90-99 Rule" for
+# two-dimensional data.
+
```

Comment:
```suggestion
```

Reply:
Hi @jklymak,
I have integrated your suggestion into the file. Looks like the linters are happy, too.
As far as I am concerned, it can stay like this until I have have found links that cover the issue well.